service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisCaseSubmission.java

```diff
@@ -26,7 +26,7 @@
     private final Allocation allocation;
     private final DocmosisHearingPreferences hearingPreferences;
     private final DocmosisInternationalElement internationalElement;
-    private final String userFullName;
+    private final String signeeName;
```

Comment:
this change requires a bit of coordination how things are released. Docmosis template has been changed to use new filed, but the problem is that release path is different for application code and docmosis.
If code is deployed first and sends signeeName then doc will be generated without name
Same situation when docmosis is deployed first and expects signeeName to be sent but old code sends userFullName
Option is to make template backward compatible for time being. It can use signeeName, but if not present then can use userFullName. Other option is to create new template e.g FL-PLW-APP-ENG-00095v2.doc which uses signeeName, point this code to new template, and release template first.

Reply:
Reverted to userFullName